cmd/system-probe/modules/network_tracer.go

```diff
@@ -78,13 +80,83 @@ type networkTracer struct {
 	tracer       *tracer.Tracer
 	done         chan struct{}
 	restartTimer *time.Timer
+
+	connectionserver.UnsafeSystemProbeServer
+	maxConnsPerMessage int
 }
 
 func (nt *networkTracer) GetStats() map[string]interface{} {
 	stats, _ := nt.tracer.GetStats()
 	return stats
 }
 
+func getConnectionsFromMarshaler(marshaler encoding.Marshaler, cs *network.Connections) ([]byte, error) {
+	buf, err := marshaler.Marshal(cs)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Tracef("GetConnections: %d connections, %d bytes", len(cs.Conns), len(buf))
+	return buf, nil
+}
+
+func min(a, b int) int {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func (nt *networkTracer) GetConnections(req *connectionserver.GetConnectionsRequest, s2 connectionserver.SystemProbe_GetConnectionsServer) error {
+	start := time.Now()
+
+	runCounter := atomic.NewUint64(0)
```

Comment:
this is a "bug" and does not match to the http version

Reply:
fixed? or I miss understood it